main.py

```diff
@@ -576,12 +576,33 @@ def get_project_path(window: sublime.Window) -> 'Optional[str]':
         return None
 
 
+def get_common_parent(paths: 'List[str]') -> str:
+    """
+    Get the common parent of multiple paths. Attempts to use native method
+    if available.
+    """
+    try:
+        return os.path.commonpath(paths)
+    except:
+        pass
+
+    groups = [s.split(os.path.sep) for s in paths]
+    min_len = min(len(group) for group in groups)
+    common = []
+    for i in range(min_len):
+        bits = set(group[i] for group in groups)
+        if len(bits) != 1:
+            break
+        common.append(bits.pop())
+    return os.path.sep.join(common)
```

Comment:
Thanks for thinking of `is_in_workspace`! I have two requests:
* We may want to change to `commonpath` if python 3.5 becomes available, but not run through a try-catch on every call. Can you make a comment about common path's existence instead?
* Your code looks very complete, but we have a simpler use case of only expecting directories as output. Can we use `dirname` on the output of `commonprefix` instead?

Reply:
> Can we use `dirname` on the output of `commonprefix` instead?

Good catch; I didn't think about this fully. I think this should work if we also append `os.path.sep` to the project path, since:
```
commonprefix(['/usr1/', '/usr2/']) -> /usr
dirname('/usr') -> '/'

commonprefix(['/usr/', '/usr/a/']) -> /usr/
dirname('/usr/') -> '/usr'
```

I'll update.

Reply:
> to the project path

(Actually, should be safe to add to all paths.)